config/config.go

```diff
@@ -225,7 +226,8 @@ type Adapter struct {
 		Password string `mapstructure:"password"`
 		Tracker  string `mapstructure:"tracker"`
 	} `mapstructure:"xapi"` // needed for Rubicon
-	Disabled bool `mapstructure:"disabled"`
+	Disabled         bool            `mapstructure:"disabled"`
+	ExtraAdapterInfo json.RawMessage `json:"extra_info,omitempty"`
```

Comment:
Can you add a test of this field to the existing `TestFullConfig` please?

Reply:
You will also need to add this to `setBidderDefaults()` so that this new field can be set from an environment variable.

Reply:
Done